rasa/nlu/selectors/response_selector.py

```diff
@@ -194,6 +197,7 @@ def __init__(
         index_label_id_mapping: Optional[Dict[int, Text]] = None,
         index_tag_id_mapping: Optional[Dict[int, Text]] = None,
         model: Optional[RasaModel] = None,
+        response_text_key_dict=None,
```

Comment:
```suggestion
        response_text_key_dict: Optional[Dict[Text, Any]] = None,
```

Reply:
It seems like the dictionary contains only Text -> Text. Let's also use that in the type definition.

Reply:
Let's also name it consistently with the other mappings we have. What about `response_retrieval_intent_mapping`?

Reply:
called it `retrieval_intent_mapping`